test/test_alias.cpp

```diff
@@ -108,6 +108,29 @@ TEST_F(AliasAnalysisTest, View_MergeNonContiguous) {
   EXPECT_EQ(alias_analysis.findRoot(out), out);
 }
 
+TEST_F(AliasAnalysisTest, Set) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  const std::vector<int64_t> in_shape({2, 3, 5});
+
+  TensorView* in = makeContigConcreteTensor(in_shape);
+  fusion.addInput(in);
+  TensorView* out = set(in);
+  fusion.addOutput(out);
```

Comment:
nitpick: can we also add a test where we have output stride set that's not consistent/compatible with the input stride and would block aliasing?

Reply:
Done. Note that AliasAnalysis itself still produces the alias with a preferred allocation domain; it's MarkAliasPass that checks the allocation domain is compatible.